src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,11 +1,25 @@
 import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react'
+import { useMemo } from 'react'
 import type { BreadcrumbsRoute } from 'react-router-breadcrumbs-hoc'
 import withBreadcrumbs from 'react-router-breadcrumbs-hoc'
+import type { RouteComponentProps } from 'react-router-dom'
 import { Link } from 'react-router-dom'
 import { AccountLabel } from 'components/AssetHeader/AccountLabel'
+import { Text } from 'components/Text/Text'
 import { selectAssetById } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
+const GetTranslatedPathPart = ({ match }: { match: RouteComponentProps['match'] }) => {
```

Comment:
```suggestion
const getTranslatedPathPart = ({ match }: { match: RouteComponentProps['match'] }) => {
```

Reply:
These are actually React functional components and their name should start with the Uppercase letters.